test/unit/config/config-env.test.js

```diff
@@ -732,4 +732,15 @@ tap.test('when overriding configuration values via environment variables', (t) =
       t.end()
     })
   })
+
+  t.test('should convert NEW_RELIC_GRPC_IGNORE_STATUS_CODES to integers', (t) => {
```

Comment:
Do you think we should add a test where the env var isn't an array? IE `NEW_RELIC_GRPC_IGNORE_STATUS_CODES: '9,5-7,blah,2'`?

Reply:
I'll just change this.